robotpy_installer/sshcontroller.py

```diff
@@ -50,10 +50,13 @@ def ssh_exec_commands(self, commands, existing_connection=False):
         if not existing_connection:
             self.ssh_connect()
 
-        _, stdout, _ = self.client.exec_command(commands)
+        _, stdout, stderr = self.client.exec_command(commands)
```

Comment:
Is there a way to print stdout/stderr in a combined manner? Otherwise, the error messages might not make a whole lot of sense.

Reply:
Do you have any ideas of what would make it better? Either way it's going to be in a weird order because of the way that it's being returned.

Reply:
Was looking at this earlier; paramiko does support having stderr on the stdout channel. I think I dumped a link to the relevant docs in the Gitter the other day.